docker/install/ubuntu_install_cmake_source.sh

```diff
@@ -20,8 +20,8 @@ set -e
 set -u
 set -o pipefail
 
-v=3.14
-version=3.14.7
+v=3.22
```

Comment:
looks like this also needs to change to get a valid URL below
```suggestion
v=3.18
```